tests/BlastDBNodeTest.php

```diff
@@ -0,0 +1,134 @@
+<?php
+namespace Tests;
+
+use StatonLab\TripalTestSuite\DBTransaction;
+use StatonLab\TripalTestSuite\TripalTestCase;
+
+/**
+ * Tests BlastDB Node CRUD
+ */
+class BlastDBNodeTest extends TripalTestCase {
+  // Uncomment to auto start and rollback db transactions per test method.
+  use DBTransaction;
+
+  /**
+   * BlastDB Node Type exists.
+   *
+   * Check that the BlastDB node type exists. It should be created
+   * when the module is installed by the Drupal Node API.
+   */
+  public function testBlastDBNodeExists() {
+
+    // Get a list of all types available.
+    $types = node_type_get_types();
+
+    // The BlastDB node type must be in the list.
```

Comment:
You can pass the assertion a third option, which is an additional message to display if the test fails.

You might consider moving your comments into assertion fail messages: that way if the test fails, the dev will know why at the console without having to read the test documentation.

Reply:
That’s awesome! I will definitely do that!